limiter.go

```diff
@@ -6,11 +6,10 @@ import (
 
 // SetRateLimiter sets the rate limiter
 func (self *Runtime) SetRateLimiter(limiter *rate.Limiter) {
-	self.limiter = limiter
 	if limiter == nil {
 		return
```

Comment:
[opt/q] would it be more intuitive to set `self.limiter` to `rate.NewLimiter(rate.Inf, maxInt)` when `limiter` is nil? That way if the default value is changed to something else, this won't still be a noop. I don't have strong feelings about this at all, so feel free to ignore.